glean-core/src/lib.rs

```diff
@@ -402,6 +412,20 @@ impl Glean {
         self.max_events
     }
 
+    /// Gets the next task for an uploader. Which can be either:
+    ///
+    /// Wait - which means the requester should ask again later;
```

Comment:
I think it would be better to refer to the `PingUploadTask` documentation.
That way we avoid duplicating this doc and we don't need to remember to update this when we change `PingUploadTask`

Reply:
Yes, I agree. If I do that docs-linkcheck also fails, when looping through the docs for glean-preview.

So, yesterday I was trying to fix the docs-linkcheck bug by making the upload module public. It did not fix it, but it did fix de Android tests I have 0 ideas why. Now that this is behing a feature flag, it should not brak anything. Still the problem should show up again later.

Reply:
for now we can refer to it without a link.

Reply:
Did you intend to still change this?

At the very least this should be a proper markdown list:

```
/// * `Wait` - ...
/// * `Upload(PingRequest)` - ...
```

Reply:
Hm, I am sorry for resolving this one and the other review comment I understood I didn't need to change anything. I now see I understood incorrectly.